src/accountsdb/db.zig

```diff
@@ -488,7 +516,16 @@ pub const AccountsDB = struct {
                 }
             }
 
-            printTimeEstimate(&timer, total_bins, iteration_count, "combining thread indexes", null);
+            if (print_progress and progress_timer.read() > DB_PROGRESS_UPDATES_NS) {
+                printTimeEstimate(
+                    logger,
+                    &timer,
+                    total_bins,
+                    iteration_count,
+                    "combining thread indexes",
+                    "thread0",
+                );
+            }
         }
```

Comment:
I don't see reset being called over here, is that intentional? Might be another place to use `lap` (as mentioned above)

Reply:
5a79870